.github/workflows/cd-sql-engine-tests.yaml

```diff
@@ -15,7 +15,7 @@ env:
 jobs:
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 6]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->With this change this workflow_dispatch trigger will succeed in doing nothing.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dbt-labs/metricflow/1253?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->